bots/images/scripts/ovirt.setup

```diff
@@ -0,0 +1,151 @@
+#!/bin/bash
+
+set -eux
+
+echo foobar | passwd --stdin root
+
+/usr/bin/systemctl stop firewalld NetworkManager kdump
+/usr/bin/systemctl disable firewalld NetworkManager kdump
+iptables -F
```

Comment:
ovirt is incompatible with both firewalld and networkmanager. Since part of the installation is bringing up of the system it needs to be disabled. It's normally done in the engine-setup but here I need to do that manually since the host part (vdsm) is installed in a non-standard way (otherwise it would require a dedicated machine and separate installation as part of "adding the host" - doable, but more lengthy both time-wise and code-wise)
kdump is just to speed things up as we do something with it in ovirt startup as well